hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/SchedulerExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.mesos.hdfs.scheduler;
+
```

Comment:
I have a concern that this handler is only 2 lines of code. A log message and exiting. I think we don't need this class at all since I'd rather keep our code to a minimum.